client/gui/Shortcut.h

```diff
@@ -0,0 +1,140 @@
+/*
+ * Shortcut.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#pragma once
+
+enum class EShortcut
+{
+	NONE,
+
+	GLOBAL_CONFIRM,    // Enter
+	GLOBAL_CANCEL,     // Escape
+	GLOBAL_RETURN,     // Enter, Escape
+	GLOBAL_FULLSCREEN, // F4
+	GLOBAL_OPTIONS,
+	GLOBAL_BACKSPACE,
+	GLOBAL_MOVE_FOCUS,
```

Comment:
can you leave a small description nearby each const what it stands for?
some of those names are not obvious

Reply:
Okay, will review naming.